src/Form/Form.js

```diff
@@ -12,15 +12,16 @@ const useStyles = makeStyles((theme) => ({
   },
 }));
 
-export default function ControlledOpenSelect({selectSemester}) {
+export default function Form({selectSemester}) {
   const classes = useStyles();
   const [semester, setSemester] = React.useState('');
   const [open, setOpen] = React.useState(false);
 
   const handleChange = (event) => {
```

Comment:
I'm glad we were able to fix the display of the dropdown bar here!

Reply:
Agreed - much more user-friendly to see the semester being queried